src/viur/core/modules/user.py

```diff
@@ -281,11 +281,10 @@ class LostPasswordStep2Skel(skeleton.RelSkel):
             required=True,
         )
 
-
     @exposed
     @force_ssl
     @skey(allow_empty=True)
-    def login(self, *, name=None, password=None, **kwargs):
+    def login(self, *, name: str = "", password: str = "", **kwargs):
```

Comment:
What makes the difference?

Both values wouldn't pass l.291 (`if not name or not password:`)

Reply:
Empty str and None is both False.
```python
>>> name = ""
>>> password = ""
>>> not name or not password
True
```

Reply:
Yes, that is known. That's why I'm asking where the difference or the concrete advantage lies. I think `None` makes more sense for the absence of a parameter than an empty string, which says that the value was sent but is wrong or empty.

Reply:
Okay. That was more a question of interest. For me, both variants would be okay.

Reply:
Would you be happier with the `str | None = None` notation?